pagopa/pn-frontend
# Fix/pn 10637: made case insensitive fiscalCode field

## Short description
Made case insensitive fiscalCode field

## List of changes proposed in this pull request
- apply toUpperCase() to fieldValue fiscalCode in newDelegation PG
- apply toUpperCase() to fieldValue fiscalCode in newDelegation PF

## How to test
Login in as PF/PG; go to Deleghe page, create a new delegation using a fiscal code with lower case characters; new delegation should be added.
